### DISCUSSION TITLE
XYChart with customized react-spring config (spring mass, tension, etc.)

### DISCUSSION THREAD

**Discussion by ThaddeusChristopher** [airbnb/visx#1482] [Q&A]
Re: XYChart

I would like to adjust the animation behavior (https://react-spring.io/common/configs) of AnimatedLineSeries and the other Animated components.

Is this possible?

And if not, would you consider exposing this functionality as a future feature?

**Comment by Cpt-Falcon** [comment]
bump

**Comment by maflaven** [comment]
I'd love to have this customizability! It's convenient that AnimatedLineSeries exists in the first place but being able to pass react-spring config like easing+duration would really enhance site uniformity (i.e., match vanilla CSS animations).
